configuration/cdap-site.xml

```diff
@@ -46,6 +57,36 @@
   </property>
 
   <property>
+    <name>master.startup.checks.enabled</name>
+    <value>true</value>
+    <description>
+      Whether checks should be run before startup to determine if the CDAP Master can be run correctly.
+      Which checks are run is determined by the master.startup.checks.packages and master.startup.checks.classes
+      settings. If any checks fail, the CDAP Master will fail to start instead of waiting for the problem to be fixed.
+      This setting only affects Distributed CDAP. It does not apply to Standalone CDAP.
+    </description>
+  </property>
+
+  <property>
+    <name>master.startup.checks.packages</name>
+    <value>co.cask.cdap.master.startup</value>
+    <description>
+      Comma-separated list of packages containing checks that will be run before the CDAP Master starts up.
+      If any of the checks fails, the CDAP Master will not start up.
+      Checks will only be run if master.startup.checks.enabled is set to true.
+    </description>
+  </property>
+
+  <property>
+    <name>master.startup.checks.classes</name>
```

Comment:
just to point out a possible inconsistency.  the omission of ``<value>`` matches cdap-default.xml but not sure if it's the right way to express null.  In the CSD it gets set to empty string since I don't think it even supports null.  So nothing to fix here now but we should keep this in mind.

 in the CSD: this gets set to empty string by default in the CSD.  This matches cdap-default.xml
just to highlight, doublecheck, is omitting the ``<value>`` element the preferred way for ambari?  I assume this is to distinguish from empty string.  But i'm pretty sure it's set to an empty string